joint_trajectory_controller/src/joint_trajectory_controller.cpp

```diff
@@ -554,6 +555,24 @@ rclcpp_action::GoalResponse JointTrajectoryController::goal_callback(
     }
   }
 
+  for (auto i = 0ul; i < goal->trajectory.joint_names.size(); ++i) {
```

Comment:
Sorry I think I've missed this part. 
You are basically checking if every joint in the goal is supported by the controller.

I see the point of the first loop but not so much with the second one. Couldn't a simple `std::find` do that for loop managing a boolean flag?

Reply:
replace with std::find d93864f

Reply:
thanks!